packages/now-routing-utils/src/superstatic.ts

```diff
@@ -108,10 +108,21 @@ function isString(key: any): key is string {
 }
 
 function replaceSegments(segments: string[], destination: string): string {
-  if (destination.includes(':')) {
+  const parsedDestination = parseUrl(destination, true);
+  let { pathname='' } = parsedDestination;
+  pathname = pathname || '';
```

Comment:
What's the point of this line?

Reply:
Was left over from suggestion, looks like TypeScript doesn't like setting the default value that way for some reason 🤔

Reply:
Looks like `pathname` can be `null` now so setting the default value that way doesn't work in all cases